__tests__/cache-worker.ts

```diff
@@ -72,7 +72,7 @@ function setUpErrorsAtApi(wrongKeys: string[], maxRetriesBeforeWorking = 0) {
 
   global.server.use(
     graphql.link(apiEndpoint).mutation('_cache', (req, res, ctx) => {
```

Comment:
```suggestion
    graphql.link(apiEndpoint).mutation<{ variables: { cacheUpdate: { keys: string[] }}}>('_cache', (req, res, ctx) => {
```

should help with the type checking (with this type parameter you set the type of body of the request = what is passed as json to the request in case `content-type: application/json`)